pinot-core/src/main/java/org/apache/pinot/core/operator/combine/BaseSingleBlockCombineOperator.java

```diff
@@ -92,6 +92,9 @@ protected void processSegments() {
           ((AcquireReleaseColumnsSegmentOperator) operator).acquire();
         }
         resultsBlock = (T) operator.nextBlock();
+      } catch (Exception e) {
+        onProcessError(operator, e);
```

Comment:
Should we consider wrapping it with exception message with segment name instead of logging? Currently it will cause duplicate log because we will log again when catching the exception

Reply:
sgtm